bin/gwdetchar-lasso-correlation

```diff
@@ -5,6 +5,11 @@
 # This file is part of the GW DetChar python package.
 #
 # GW DetChar is free software: you can redistribute it and/or modify
+#it under terms of Copyright (C) LIGO Scientific Collaboration (2015-)
+#
+# This file is part of the GW DetChar python package.
+#
+# GW DetChar is free software: you can redistribute it and/or modify
```

Comment:
@macedo22, was this duplication intentional? either way it should be removed...

Reply:
no, that must have happened by accident when pasting and I didn't catch it. would you like me to add a new commit?

Reply:
actually, I believe a typo in my version caused it to merge strangely when I rebased

Reply:
@macedo22, yes, please correct this with a new commit.